thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/views/AnomalyResultView.js

```diff
@@ -225,36 +225,27 @@ AnomalyResultView.prototype = {
     });
 
   },
+
   renderAnomaliesTab : function(anomaliesWrapper) {
-    var anomalies = anomaliesWrapper.anomalyDetailsList;
-    for (var idx = 0; idx < anomalies.length; idx++) {
-      var anomaly = anomalies[idx];
+    const anomalies = anomaliesWrapper.anomalyDetailsList;
+    for (let idx = 0; idx < anomalies.length; idx++) {
```

Comment:
Why not use Array#forEach to iterate the anomalies? forEach is declarative, hence clearer and more readable than an imperative for loop. You can also simulate a `continue` in a forEach by replacing it with a `return` statement:

````javascript
if (!anomaly) {
  return;
}
````
Also you won't need to manually cache each anomaly in a local var